language/de-DE/tpl_cassiopeia.ini

```
@@ -8,6 +8,7 @@ TPL_CASSIOPEIA_XML_DESCRIPTION="Weiterentwicklung der Space-Themes (Solarflare i
 
 CASSIOPEIA="Cassiopeia Frontend Template"
 TPL_CASSIOPEIA_BACKTOTOP="Zurück nach oben"
+TPL_CASSIOPEIA_BACKTOTOP_LABEL="Link „Zurück nach oben“"
```

```suggestion
TPL_CASSIOPEIA_BACKTOTOP_LABEL="„Zurück nach oben“ Link"
```
sounds better @zero-24 ?

---

Agree.